TELEMETRY/mAhRmn/mahRe2.lua

```diff
@@ -132,6 +134,9 @@ local GV = {[1] = 0, [2] = 1, [3] = 2,[4] = 3,[5] = 4,[6] = 5, [7] = 6, [8] = 7,
 --
 -- Change as desired
 -- Use GV[6] for GV6, GV[7] for GV7 and so on
+
+local GVCapResPer = GV[6] -- Used to manually set the remaining battery capacity (percent) (sphrases)

```

Comment:
Could you add and test
local ReadGVCapResPer = false -- set to true to read reserver form a global variable and have different reserves per model